src/Listeners/BluehostPlugin.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace NewfoldLabs\WP\Module\Data\Listeners;
+
+/**
+ * Monitors generic plugin events
+ */
+class BluehostPlugin extends Listener {
```

Comment:
Since we're adding this back in here, should we rename it to `BrandPlugin`? That or should this live in the bluehost plugin specifically and each brand plugin can have their own hooks with their own logic?

Reply:
There aren't any event listeners that I see in those other plugins. 2 of the 3 aren't actually Bluehost specific (anymore). Until we get Mojo and Bluehost using the latest modules, we'll need to handle these in a central location like the data module. Yes, it is temporary, and I've already created some tasks around getting these moved back out later: https://github.com/newfold-labs/wp-module-data/issues